src/pages/use-cases/network-policy.jsx

```diff
@@ -127,6 +128,18 @@ const testimonials = [
     description:
       'Cilium network policies can secure communication between the Kubernetes API server and the kubelet by assigning them identities and using a CiliumNetworkPolicy to ensure that only traffic from the API server is allowed to reach the kubelet. eBPF provides an efficient means to enforce network policies for host processes. Cilium utilizes eBPF programs that attach to network interfaces to identify the source of network traffic. These programs insert the identity of the sending process into the packet header, which the receiving host can use to determine whether to allow the traffic.',
   },
+  {
+    logo: GDataLogo,
+    title: 'Migrating to Cilium for Better Networking, Visibility and Security',
+    CTAs: [
+      {
+        CTAtext: 'Read Case Study',
+        url: 'https://www.cncf.io/case-studies/g-data-cyberdefense/',
+      },
+    ],
+    description:
+      'G DATA chose Cilium as their preferred solution for networking, observability, and security to benefit from its enhanced network policy visibility. Cilium allows their development teams to self service network policies without needing to involve the IT team.',
```

Comment:
can you use this quote instead?

In the beginning, it was hard for our developers to write network policies because we were in our early Kubernetes adoption phase. Everyone had to learn a lot of stuff in Kubernetes and then also had to learn how to write network policies. Cilium helped reduce the mental overhead and helped speed up our development process so that we can bring new features to customers faster.”

Reply:
Alright